src/Github/Api/GraphQL/Query/GetMilestoneFirst100IssuesAndPullRequests.php

```diff
@@ -13,77 +13,74 @@ final class GetMilestoneFirst100IssuesAndPullRequests implements GetGithubMilest
 {
     // @TODO this fetches ONLY the first 100 issues!!!
     private const QUERY = <<<'GRAPHQL'
-query GetStuff($owner: String!, $repositoryName: String!, $milestoneNumber: Int!) {
-  repository(name: $repositoryName, owner: $owner) {
-    milestone (number: $milestoneNumber) {
-      url,
-      number,
-      closed,
-      title,
-      description,
-      url,
-      issues (first: 100) {
-        nodes {
-          number,
-          title,
-          closed,
-          url,
-          author {
-            login,
-            url
-          },
-          labels (first: 100) {
-            nodes {
-              color,
-              name,
-              url
+        query GetStuff($owner: String!, $repositoryName: String!, $milestoneNumber: Int!) {
```

Comment:
Would probably keep this as it was before: I don't trust anything in GraphQL, after some bad experiences with it :D

Reply:
Ah, wait, did the indentation change, or just the nowdoc wrapper?

Reply:
Just the indent